CMakeLists.txt

```diff
@@ -11,16 +11,31 @@ project(bpfilter
 
 include(GNUInstallDirs)
 
-find_package(Doxygen REQUIRED)
+option(BUILD_TESTING "Enable testing" OFF)
+option(BUILD_DOCS "Enable building documentation" OFF)
```

Comment:
Those should be enabled by default to keep the current behaviour. Use `WITH_XXX` instead of `BUILD_XXX` similarly to #174.